faucet-server/Sources/App/Controllers/CKBController.swift

```diff
@@ -61,13 +67,67 @@ struct CKBController: RouteCollection {
     }
 
     func makeRandomAddress(_ req: Request) -> Response {
-        let privateKey = CKBService.generatePrivateKey()
+        let privateKey = generatePrivateKey()
         let result: [String: Any] = [
             "privateKey": privateKey,
-            "publicKey": try! CKBService.privateToPublic(privateKey),
-            "address": try! CKBService.privateToAddress(privateKey)
+            "publicKey": try! privateToPublic(privateKey),
+            "address": try! privateToAddress(privateKey)
         ]
         let headers = HTTPHeaders([("Access-Control-Allow-Origin", "*")])
         return Response(http: HTTPResponse(headers: headers, body: HTTPBody(string: result.toJson)), using: req.sharedContainer)
     }
+
+    // MARK: - Utils
+
+    public func faucet(address: String) throws -> H256 {
```

Comment:
What does `faucet` mean here? This function name is very confusing.

Reply:
It seems it's a transfer method, and I think it's better to move out of controllers, and amount of capacity is better to be a config instead of hardcode.

BTW, faucet use `asw` or default lock ?

Reply:
I've used environment parameters to set the send capacity count

Reply:
The main code for `sendCapacity` is in `services/wallet/wallet.swift`, which is just a call

Reply:
I don't understand, but CellOutput lock use `system script`

Reply:
@classicalliu